packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -246,26 +249,28 @@ export default class QuanticSearchBoxInput extends LightningElement {
   }
 
   onTextAreaInput() {
-    this.handleValueChange();
+    this.sendInputValueChangeEvent(this.input.value, true);
     this.adjustTextAreaHeight();
   }
 
   adjustTextAreaHeight() {
     if (!this.textarea) {
       return;
     }
-    this.input.value = this.input.value.replace(/\n/g, '');
     this.input.style.height = '';
     this.input.style.whiteSpace = 'pre-wrap';
     this.input.style.height = this.input.scrollHeight + 'px';
+    this.input.style.overflow = 'auto';
```

Comment:
Sure, actually `overflow = 'auto'` is the natural behaviour so it shouldn't change anything.
This is added just because I'm adding `this.input.style.overflow = 'hidden';` when collapsing the expandable search box.


https://github.com/coveo/ui-kit/assets/86681870/858365d0-19f4-4aab-9ab8-2d5602bae672